resident/resident-service/src/main/java/io/mosip/resident/service/impl/ResidentCredentialServiceImpl.java

```diff
@@ -550,6 +603,28 @@ private NotificationResponseDTO sendNotification(String id,
 				additionalAttributes);
 		return notificationService.sendNotification(notificationRequest);
 	}
+	
+	@SuppressWarnings("unused")
+	private void insertCredentialreqInDB(boolean isCredReqSuccess, String purpose,
+			String credentialRequestId) throws ResidentServiceCheckedException, NoSuchAlgorithmException, ApisResourceAccessException{		
+		ResidentTransactionEntity residentTransactionEntity = new ResidentTransactionEntity();
+		LocalDateTime now = LocalDateTime.now();
+		residentTransactionEntity.setEventId(UUID.randomUUID().toString());
+		residentTransactionEntity.setPurpose(purpose);
+		residentTransactionEntity.setRequestDtimes(LocalDateTime.now());
```

Comment:
Make all LocalDateTime.now() calls withing this method assigned to one variable.